### PR TITLE
add support for document upload

### PR DESCRIPTION
**Pull request by tsiq-chandru**
* added post on case/<CASE ID>/documents
* added get on case/<CASE ID>/documents

### CODE REVIEW

File: claims/adapters.py [hunk 1]
```diff
@@ -103,15 +106,19 @@ def set_field(self, case_id, field, obj):
         """ We use the description field as a JSON blob that stores fields
```
Comment (MykolaGolubyev) [inline claims/adapters.py]: is CLAIM_JSON_DATA new custom field? If so may need to update this comment

File: claims/adapters.py [hunk 2]
```diff
@@ -63,34 +64,36 @@ def _case_from_ticket(self, ticket):
...
+
     def get_case_by_id(self, case_id):
         return self._case_from_ticket(self._get_ticket_by_case_id(case_id))
 
     # todo: there is a lot of repeated set/get methods -- generalize
     # todo: write tests after generalization
     def set_reserve(self, case_id, reserve):
-        ticket = self._get_ticket_by_case_id(case_id)
-        issue = self.client.issue(ticket.key)
+        # ticket = self._get_ticket_by_case_id(case_id)
```
Comment (MykolaGolubyev) [inline claims/adapters.py]: is this a temp comment?

File: claims/adapters.py [hunk 3]
```diff
@@ -103,15 +106,19 @@ def set_field(self, case_id, field, obj):
         """ We use the description field as a JSON blob that stores fields
         """
         ticket = self._get_ticket_by_case_id(case_id)
-        raw = json.loads(ticket.fields.description)
+        data = self._get_custom_attr(ticket, CLAIM_JSON_DATA)
+        raw = json.loads(data)
+        # raw = json.loads(ticket.fields.description)
```
Comment (MykolaGolubyev) [inline claims/adapters.py]: seems like left here by accident

File: claims/api/swagger.yml [hunk 4]
```diff
@@ -276,12 +281,63 @@ paths:
...
+      parameters:
+        - name: aCaseId
+          in: path
+          type: string
+          required: true
+        - name: label
+          in: formData
+          description: A label for this file
+          type: string
+        - name: theFile
```
Comment (MykolaGolubyev) [inline claims/api/swagger.yml]: will this allow to attach multiple files at once?

Comment (tsiq-chandru) [reply claims/api/swagger.yml]: not at this moment -- is that a requirement to do batch file loads?

Comment (tsiq-nick) [reply claims/api/swagger.yml]: on UI users can drag multiple files and then press Upload. We can change to one at a time. Multifile support should be part of the multipart form-data out of the box

**Pull request merged**